README.md

```diff
@@ -1,18 +1,334 @@
-# Changing The DOM with DevTools and JavaScript
+# JavaScript Fundamentals: JavaScript Variables
 
 ## Learning Goals
 
-1. Target nodes with JavaScript
-2. Update/delete nodes with DevTools
+1. Define a variable
+2. Name variables in JavaScript
+3. Initialize variables in JavaScript
+4. Identify when to use `const`, `let`, and `var` for declaring variables
 
 ## Introduction
 
-In the previous lab we saved some information into a variable. "Saving" to a variable allows us to "save" a result so we can use it again later.  Storing calculations to "temporary storage places" is the heart of making efficient programs. It's a simple idea that has powerful consequences. But for the moment, let's look at the options we have when it comes to storing information.
+If you've noticed previously, we saved some information into a variable.
```

Comment:
I'd tighten this up to just "Previously, we saved some information into a variable."